stable/kubernetes-dashboard/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.rbac.create }}
```

Comment:
I think there may be more logic needed here. I was just testing this and ran into a case when `.Values.rbac.create .Values.rbac.clusterReadOnlyRole` are both true, the above role and this role are both created have the exact same name.

I noticed the previous chart used an else statement to achieve this but not sure if you had to break it out for another specific reason (I don't have a ton of time atm to dig into it but figured I'd point it out):
https://github.com/helm/charts/pull/15744/files#diff-037758159ff89f7c32f0795c811f9d06L26

Either way, great work! This worked great in our test cluster when disabling the read only role.

Reply:
Thanks! You were right, something went wrong during my first attempt to convert all of this to more sane way of managing RBACs. This should be fixed in the last commit.